ElevatorSim/src/elevatorsim/constants/ElevatorState.java

```diff
@@ -0,0 +1,52 @@
+package elevatorsim.constants;
+
+import elevatorsim.common.ElevatorStatus;
+import elevatorsim.common.SerializableMessage;
+
+/**
+ * Represents the different states the elevator can be in
+ * Note: The elevator must not be moving if it is in the door closed state (or other states starting in door for that matter)
+ * @author Trevor Bivi
+ */
+public enum ElevatorState {
+	DOOR_OPEN ((byte) 0x01, "Door open"),
```

Comment:
DOOR_OPEN and DOOR_CLOSE have the same byte values

Reply:
fixed